.eslintrc.js

```diff
@@ -20,6 +20,8 @@ module.exports = {
     'no-unused-vars': 0,
     'no-extra-boolean-cast': 0,
     'no-inner-declarations': 0,
+    'no-constant-condition': 0,
```

Comment:
I think this rule can be disabled one time. No need to disable this for the whole project